app/workers/subject_set_subject_counter_worker.rb

```diff
@@ -0,0 +1,19 @@
+class SubjectSetSubjectCounterWorker
+  include Sidekiq::Worker
+
+  sidekiq_options congestion: {
+    interval: 1.hour,
+    max_in_interval: 10,
+    min_delay: 3.minutes,
+    reject_with: :cancel,
```

Comment:
I meant a test just to see that the reschedule/cancel works like we'd want in case of congestion.